config/rollup.ses.config.js

```diff
@@ -0,0 +1,31 @@
+import config from './rollup.iife.config';
+import fs from "fs";
+import { builtinModules as builtin } from "module";
+import jscc from 'rollup-plugin-jscc';
+import replace from "@rollup/plugin-replace";
+
+const pkg = JSON.parse(fs.readFileSync("./package.json"));
+delete pkg.dependencies["ejs"];
```

Comment:
have to remove `ejs`, because this line is not SES compatible: https://github.com/mde/ejs/blob/main/lib/ejs.js#L109

Reply:
That shouldn't be how this works. These are marking the dependency as "external" so it isn't put into the bundle. When you delete it from this, it will actually be included.

Reply:
this worked us because it removed the `var ejs = require('ejs');` from `main.ese.cjs`.
and the metamask uses browserify to package all this.
but yeah, we should talk before you merge any of my PRs.